InnerEye/ML/utils/ml_util.py

```diff
@@ -75,19 +75,22 @@ def string_to_integer_list(csv_list: str, length: int) -> List[int]:
     return integer_list
 
 
-def validate_dataset_paths(dataset_path: Path = Path.cwd()) -> None:
+def validate_dataset_paths(
+        dataset_path: Path = Path.cwd(),
+        dataset_csv: str = DATASET_CSV_FILE_NAME) -> None:
     """
-    Validates that a dataset.csv file exists in the given path.
+    Validates that tdhe required dataset csv file exists in the given path.
```

Comment:
typo `tdhe`

Reply:
Thanks!  Typo corrected in latest commit.